### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by bumbeishvili** [bumbeishvili/coronavirus.davidb.dev#1]
## Translation 
 words are translated using google translate, only the following languages are checked

* [x] Italy
* [x] Germany
* [x] Spain
* [x] France
* [x] Switzerland
* [x] UK
* [x] Netherlands
* [x] Austria
* [x] Belgium
* [x] Norway
* [x] Sweden
* [x] Portugal
* [x] Denmark
* [x] Turkey
* [x] Czech Republic
* [x] Ireland
* [x] Poland
* [x] Greece
* [x] Japan
* [x] China
* [x] South Korea
* [x] Canada
* [x] USA
* [x] Argentina
* [x] Brasil
* [x] South Africa
* [x] Australia

**Comment by ghost** [comment]
Hi, help me. https://ncov2019.gq

**Comment by jeffmat5** [comment]
Is it possible to add filter to country?

**Comment by jmsche** [comment]
Hi, my country is correctly identified (Belgium) but the locale seems to be set to Dutch.

The problem is, I don't speak Dutch :p Any way to add a button to switch locale? Or did I miss it?

Also it's weird as my browser's `ACCEPT_LANGUAGE` is set to `fr,fr-FR;q=0.8,en-US;q=0.5,en;q=0.3`.

Thanks!

**Comment by bumbeishvili** [comment]
Hello @jmsche , I did a little research before I set lang to dutch, this is a  fragment from Wikipedia

![Screen Shot 2020-03-25 at 00 22 43](https://user-images.githubusercontent.com/6873202/77473489-fff4b680-6e2e-11ea-9f19-3b2d648dd67e.png)

I made my decision based on this, language is set based on location, not based on locale 

Currently, there is no settings in the interface to achieve this.

What you can do is following
  open console and type

 `localStorage.setItem('language','fr')`

and press enter. If you want English as a default language, execute this option

`localStorage.setItem('language','US')`

**Comment by jmsche** [comment]
Well sure, 58% of Belgians speak dutch, but 41% speak French, hence my initial question.

Of course I can set the language this way on my desktop, but how should I do that on my smartphone? :)

Is there any way you rely on browser ACCEPT_LANGUAGE header?

**Comment by bumbeishvili** [comment]
@jmsche  yes there is way, but I don't like it,  since it won't fit my intentions 

For example - that header for my case is set to 
![Screen Shot 2020-03-25 at 00 55 22](https://user-images.githubusercontent.com/6873202/77476103-5cf26b80-6e33-11ea-8a49-27077b7524d8.png)

I want the third one to be the default language for this site, so does most people in my country.

I think the most optimal solution would be to add language switcher in the interface, default language will be set in some way, but you will have option to choose a different language

**Comment by bumbeishvili** [comment]
BTW, this platform is front end only, it does not have dedicated back end

**Comment by ghost** [comment]
My is not working
ncov2019.gq
